src/sentry/static/sentry/app/components/group/pluginActions.tsx

```diff
@@ -77,7 +75,7 @@ class PluginActions extends React.Component<Props, State> {
     });
   };
 
-  loadPlugin = data => {
+  loadPlugin = (data: any) => {
```

Comment:
can we add a type here?

Reply:
I don't mind this being any since plugin actions are v old. I think it may also be hard to add types for it.

Reply:
I wouldn't worry about typing this tbh